code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1215,7 +1215,7 @@
 					H.adjustOxyLoss(8)
 				H.throw_alert("too_much_co2", /obj/screen/alert/too_much_co2)
 			if(prob(20)) // Lets give them some chance to know somethings not right though I guess.
-				spawn(0) H.emote("cough")
+				H.emote("cough")
```

Comment:
So this one has been confusing the fuck out of me, WHY THE HELL DID SO MANY THINGS SPAWN FOR EMOTE?